pkg/mkimage-raw-efi/make-raw

```diff
@@ -205,8 +205,14 @@ do_rootfs() {
         esac
     fi
 
-    # Copy rootfs to image A
-    dd if=$IMG of=$IMGFILE bs=1M conv=notrunc seek="$(( SEC_START * 512 ))" oflag=seek_bytes
+    if [ -n "$IMG" ]; then
+      # Copy rootfs
+      dd if="$IMG" of="$IMGFILE" bs=1M conv=notrunc seek="$(( SEC_START * 512 ))" oflag=seek_bytes
+    else
+      # clean first bytes of partition
+      # to avoid keeping of stale data
+      dd if=/dev/zero of="$IMGFILE" bs=512 count=1 conv=notrunc seek="$(( SEC_START * 512 ))" oflag=seek_bytes
```

Comment:
you still use bs=512, so multiplication and "seek_bytes" flag are excessive. but this is minor

Reply:
Good point, thank you, modified.